components/options/Options.tsx

```diff
@@ -0,0 +1,130 @@
+      });
+
+    /*useEffect(() => {
+      async function getUsers(){
+        setUsers(await getUser())
+      } 
+      getUsers();
+    },[])
+
+    const editUser = async () => {
+        const response = await axios.put("http://localhost:8080/api/users/62a6f3a85e59b4912e234457", {username:nameRef.current.value.trim()}, {headers:{'Content-Type': 'application/json',
+        'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZG1pbiI6dHJ1ZSwiZXhwIjoxNjU1MTI4MzM2LCJ1c2VybmFtZSI6ImZvbyJ9.RmKj9VTsxjMR1JnMHaE4bO6wbMF8tCoGt7e3EterseU'}
+        })
+    }
+    
+    const getUser = async () => {
+        const response = await axios.get("http://localhost:8080/api/users/62a6f3a85e59b4912e234457", {headers:{'Content-Type': 'application/json',
+        'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZG1pbiI6dHJ1ZSwiZXhwIjoxNjU1MTI4MzM2LCJ1c2VybmFtZSI6ImZvbyJ9.RmKj9VTsxjMR1JnMHaE4bO6wbMF8tCoGt7e3EterseU'}
+        })
+        return response.data
+    }
+
+    const deleteUser = async () => {
+        const response = await axios.delete("http://localhost:8080/api/users/62a5ef5f9b388bcfc925fcdc", {headers:{'Content-Type': 'application/json',
+        'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZG1pbiI6dHJ1ZSwiZXhwIjoxNjU1MTI4MzM2LCJ1c2VybmFtZSI6ImZvbyJ9.RmKj9VTsxjMR1JnMHaE4bO6wbMF8tCoGt7e3EterseU'}
+        })
+    }
+
+    console.log(getUser());*/
+
+    const onDeleteClick = async () => {
+        try {
+          const response = await OptionsAPI.delete(currentItem._id);
+          response.status === 204
+            ? alert("User successfully deleted!")
+            : alert("Uups! Something went wrong!");
+        } catch (error) {
+          console.log(error);
+        }
+      };
+  
+    return (
+        <>
+            <div className="d-flex flex-row">
+                <div>
+                    <h1></h1>
+                    <p>
+                        <button className="btn btn-lg btn-primary"  type="button" data-bs-toggle="collapse" data-bs-target="#edit" aria-expanded="false" aria-controls="edit">
+                            Daten ändern
+                        </button>
+                    </p>
+                    <div className="collapse" id="edit">
+                        <div className="card card-body" margin-left="0 auto">
+                            <label>Name { }</label>
+                            <input
+                                type="text"
+                                className="form-control"
+                                defaultValue={user.username}
+                                ref={nameRef}>
+                            </input>
+                            <div>Email { }</div>
+                            <input
+                                defaultValue={email}
+                            ></input>
+                            <div>Passwort { }</div>
+                            <input
+                                type="text"
```

Comment:
bitte überall ändern
```suggestion
                                type="password"
```